projects/hslayers/src/components/core/event-bus.service.ts

```diff
@@ -131,10 +134,8 @@ export class HsEventBusService {
   /**
    * Fires when composition is downloaded from server and parsing begins
    * replaces 'compositions.composition_loading'
-   * @deprecated
    */
-  compositionLoading: Subject<any> = new Subject();
-  currentComposition: BehaviorSubject<any> = new BehaviorSubject(null);
+  compositionLoading: Subject<string> = new Subject();
```

Comment:
This does not hold. The only occasion where compositionLoading is broadcasted is from https://github.com/hslayers/hslayers-ng/blob/0ad079b45c3bcff0b67f5543d4a24d35c0f52f1e/projects/hslayers/src/components/save-map/save-map-manager.service.ts#L239 where its content ("compInfo") is not a string, but an object. The type-checker does not produce an error only because it is retyped to `any`.
Not sure what to do with this Subject. Perhaps delete it as well? We do not subscribe to it anywhere...

Reply:
Thanks! I was testing few possible solutions while trying to get current currentCompositionRecord to work and just missed this one.
Will get rid of this one.

Reply:
Thanks, merging.